retirement/serializers.py

```diff
@@ -558,32 +566,44 @@ def update(self, instance, validated_data):
                     # coupon on the new orderline created. In other words, any
                     # coupon used for the initial purchase is applied again
                     # here.
-                    need_transaction = True
-                    amount = (
-                        validated_data['retreat'].price -
-                        order_line.coupon_real_value
-                    )
-                    if not (payment_token or single_use_token):
-                        raise serializers.ValidationError({
-                            'non_field_errors': [_(
-                                "The new retreat is more expensive than "
-                                "the current one. Provide a payment_token or "
-                                "single_use_token to charge the balance."
-                            )]
-                        })
+                    # need_transaction = True
+                    # amount = (
+                    #     validated_data['retreat'].price -
+                    #     order_line.coupon_real_value
+                    # )
+                    # if not (payment_token or single_use_token):
```

Comment:
No need to keep commented code without any explanation

Reply:
code removed and linked to git commit